docs/content/docs/updates/v1.0.0.md

```diff
@@ -6,12 +6,39 @@ summary: |
   The 1.0 release is the first major release of the Fluid Framework.
 ---
 
-## Patch Updates
+# 1.3.0
 
-### Update 1.0.1:
+## @fluidframework/fluid-static
+
+We are now surfacing errors on FluidContainer “disposed” event, so the application can appropriately react to faulty conditions, if container was closed due to an error.
+
+## Notable bug fixes
+
+- If a single AzureClient instance is used to open multiple documents/containers, the internal in-memory cache was not handling that scenario correctly, so the "latest" snapshots would be polluted by the first document loaded. The result of that would be container closing as "corrupted". This has been fixed.
+
+- Fix url deps and add eslint rules to not allow core nodejs modules on lts.
+
```

Comment:
Thoughts around how we see changes intended for partner teams (say FRS) communicated here? We have @fluidframework/azure-end-to-end-tests published and intended for partner/frs consumption (as well as our own test scenarios)

Reply:
This is a good question. Let's focus this ff.com release notes around our azure consumer release notes. 

I'm thinking we can use discussion announcements to publish notes specific to other partners.

Reply:
Will look for a good place to document some of these points.